scripts/LV/fillHouse.sqf

```diff
@@ -131,7 +150,51 @@ _fnc_notInString = {
 	private _notInString = TRUE;
 	{
 		if (toLower _type find _x >= 0) exitWith { _notInString = FALSE };
-	} forEach [ "_story", "_vr", "competitor", "ghillie", "miller", "survivor", "crew", "diver", "pilot", "rangemaster", "uav", "unarmed", "officer", "ugv" ];
+	} forEach [ 
+		"_story", 
+		"_vr", 
+		"competitor", 
+		"ghillie", 
+		"miller", 
+		"survivor", 
+		"crew", 
+		"diver", 
+		"pilot", 
+		"rangemaster", 
+		"uav", 
+		"unarmed", 
+		"officer", 
+		"ugv", 
+		"vn_o_men_nva_65_35",
+		"vn_o_men_nva_65_36",
+		"vn_o_men_nva_65_37",
+		"vn_o_men_nva_65_38",
+		"vn_o_men_nva_65_39",
+		"vn_o_men_nva_65_40",
+		"vn_o_men_nva_37",
+		"vn_o_men_nva_38",
+		"vn_o_men_nva_39",
+		"vn_o_men_nva_40",
+		"vn_o_men_nva_41",
+		"vn_o_men_nva_42",
+		"vn_b_men_army_13",
+		"vn_b_men_army_14",
+		"vn_b_men_army_23",
+		"vn_b_men_army_24",
+		"vn_b_men_army_25",
+		"vn_b_men_army_26",
+		"vn_b_men_army_28",
+		"vn_b_men_army_29",
+		"vn_i_men_army_13",
+		"vn_i_men_army_14",
+		"vn_i_men_army_23",
+		"vn_i_men_army_24",
+		"vn_i_men_army_25",
+		"vn_i_men_army_26",
+		"vn_i_men_ranger_13",
+		"vn_i_men_ranger_14",
+		"vn_i_men_ranger_22"
```

Comment:
Might have to add more to filter out. Some story characters are still in because they look ok.